docs/tutorials/substreams/substreams-app-example.experimental.md

```diff
@@ -0,0 +1,215 @@
+---
+title: 'Substreams application example'
+
+order: -9999
+outline: [0,4]
+---
+
+# Substreams Application Example
+
+If you follow the deployment of `substreams charts` from [Getting Started](getting-started.experimental.md), you should now have a working `Substreams` cluster locally.
+
+However, this local cluster will only help you to extract block data from `producer` pod or a blockchain if you connect to a public endpoint instead.
+
+To fully make use of these block data you will need something called `Substreams Packages` which will be used like a filter to extract exactly the data you need. For example:
```

Comment:
At this point I think some explanations about Substreams package is needed, such that 
it  consists of web assembly modules written in Rust and the configurations to use modules 
because there comes Node.js packages in the following, which is rather confusing.

Reply:
I'm adding a short explanation and a link to https://substreams.streamingfast.io/documentation/consume/packages